packages/bpk-component-datepicker/readme.md

```diff
@@ -15,6 +15,7 @@ import React, { Component } from 'react';
 import BpkDatepicker from 'bpk-component-datepicker';
 import format from 'date-fns/format';
 
+const formatDate = date => format(date, 'DD/MM/YYYY');
```

Comment:
that was used in the example but the declaration was missing, probably because is copied from calendar where this is not needed